### PR TITLE
doc: add 1.9.1 to main

### PR DESCRIPTION
**Pull request by greg-fer**
Added the 1.9.1 release doc updates to main.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -341,6 +342,14 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
+.. rst-class:: v1-9-0
+
+KRKNWK-13059: Wrong MAC frame counter is reported sometimes
+  The reporting of the wrong MAC frame counter causes the neighbor to drop subsequent frames from the device due to security checks.
+  This issue only affects to Thread 1.2 builds.
+
+  **Workaround:** To fix the issue, cherry-pick the commit with the hash ``1ab6be252335ceec5a966b36fbc79883ebd1c4d1``.
```
Comment (edmont) [inline doc/nrf/known_issues.rst:351]: Need to specify that this is a `sdk-zephyr` commit?

**Review by umapraseeda — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst:351]: I thougth you gave me the commit hash to `sdk-nrf`...

**Review by ilhanates — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst]: Updated as per conversation on Teams.

**Review by edmont — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -2031,6 +2040,19 @@ NCSDK-14015: Execution halts during boot
 
   **Workaround:** Place the ``rpmsg_nrf53_sram`` partition inside the ``sram_nonsecure`` partition using :ref:`partition_manager`.
 
+<<<<<<< HEAD
+=======
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:2044]: These should be removed.
```suggestion
```

Comment (greg-fer) [reply doc/nrf/known_issues.rst:2044]: What the... thanks!

Comment (greg-fer) [reply doc/nrf/known_issues.rst]: Forgot to push my edits. Please recheck.

**Review by jfischer-no — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst]: LGTM. Thanks!

**Pull request merged**